src_assets/common/assets/web/footer.html

```diff
@@ -0,0 +1,10 @@
+<script src="/node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
Just a question... but is there a functional difference between these being here at the end of the html or in the `head` section as they were before?

Reply:
Good question, older practices would put scripts in the head and then you'd do some sort of an 'on ready' to run the scripts only after all the content is loaded. By putting them at the end, it's more modern practice and helps with the loading order. https://levelup.gitconnected.com/all-about-script-87fea475b976